src/auditlog/diff.py

```diff
@@ -48,6 +49,34 @@ def get_fields_in_model(instance):
     return instance._meta.fields
 
 
+def get_field_value(obj, field):
+    """
+    Gets the value of a given model instance field.
+    :param obj: The model instance.
+    :type obj: Model
+    :param field: The field you want to find the value of.
+    :type field: Any
+    :return: The value of the field as a string.
+    :rtype: str
+    """
+    if isinstance(field, DateTimeField):
+        # DateTimeFields are timezone-aware, so we need to convert the field
+        # to its naive form before we can accuratly compare them for changes.
+        try:
+            value = field.to_python(getattr(obj, field.name, None))
+            if value is not None:
```

Comment:
Yes, that would be possible, I already noted that in the comment above. I didn't choose to do that because in the future there might be other fields that do not throw an `ObjectDoesNotExist` exception, but that do need to be distinguished using an `isinstance`-check. If you think moving the try-catch is a better solution, that's fine too and I'll change it.